packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -112,6 +113,13 @@ const INDIVIDUAL_ACTIONS = [
     label: _('forceUpgrade'),
     level: 'primary',
   },
+  {
+    collapsed: true,
+    handler: proxy => openTunnelOnProxy(proxy),
```

Comment:
That's what I did at first but I think it's important to explicitly not pass the user data (2nd param of the handler). I think I recall a few times when we introduced bugs because we weren't expecting a 2nd argument that was passed by doing that.